cpp/include/logistic_regression.h

```diff
@@ -73,7 +71,7 @@ class LogisticRegression {
   /// 
   static Vector<T> Fit(const Matrix<T> &xin, const Vector<T> &y, int iterations, T alpha){
```

Comment:
You should rename xin to x and just use it directly in the function instead of having another x declared inside the function.